modules/nf-core/mixcr/analyze/main.nf

```diff
@@ -0,0 +1,64 @@
+process MIXCR_ANALYZE {
+    tag "$meta.id"
+    label 'process_high'
+
+    secret 'MI_LICENSE'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/7d/7dccca2db544d708a462b2ba49117b41e32141d8eb46f4dbc8730e210d2cdbae/data':
+        'community.wave.seqera.io/library/mixcr:4.7.0--bb57944ca92aeb74' }"
+
+    containerOptions "${ workflow.containerEngine == 'singularity' ?
+        '-B \$HOME' :
+        '' }"
+    
+    input:
+    tuple val(meta), path(reads)
+    val preset
+    val species
+
+    output:
+    tuple val(meta), path("*clones*.tsv")    , emit: clones
+    tuple val(meta), path("*.txt")           , emit: reports, optional: true
+    tuple val(meta), path("*.contigs.clns")  , emit: clns, optional: true
+    tuple val(meta), path("*.extended.vdjca"), emit: vdjca, optional: true
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def java_mem = task.memory ? "-Xmx${task.memory.toGiga()}g" : ''
```

Comment:
The entire memory amount from task is not available as heap size and this could lead to the tool dying with out of memory before it starts doing garbage collection seriously.